<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSettings</class>
 <widget class="QWidget" name="AudioSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
   <item row="2" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Equalizer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="restoreDefaultEQ">
     <property name="text">
      <string>Reset EQ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QSpinBox" name="decay">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="audio_device">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Normalization offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="amplitude">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="avg_mode"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Filter constant</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Normalization scale</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Decay (% per step)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="avg_size">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>AudioDevice</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>FFT Window mode</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="nrml_ofst">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Average mode</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="window_mode"/>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="filter_constant">
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="nrml_scl">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Average size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QFrame" name="eq">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>96</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="11">
       <widget class="QTooltipedSlider" name="eq_12">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTooltipedSlider" name="eq_01">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QTooltipedSlider" name="eq_07">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QTooltipedSlider" name="eq_09">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTooltipedSlider" name="eq_03">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QTooltipedSlider" name="eq_14">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QTooltipedSlider" name="eq_15">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTooltipedSlider" name="eq_04">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QTooltipedSlider" name="eq_11">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QTooltipedSlider" name="eq_08">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QTooltipedSlider" name="eq_06">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTooltipedSlider" name="eq_02">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QTooltipedSlider" name="eq_13">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QTooltipedSlider" name="eq_10">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QTooltipedSlider" name="eq_05">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QTooltipedSlider" name="eq_16">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="restoreDefaultSettings">
     <property name="text">
      <string>Restore default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTooltipedSlider</class>
   <extends>QSlider</extends>
   <header location="global">QTooltipedSlider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>restoreDefaultSettings</tabstop>
  <tabstop>audio_device</tabstop>
  <tabstop>amplitude</tabstop>
  <tabstop>avg_size</tabstop>
  <tabstop>decay</tabstop>
  <tabstop>nrml_ofst</tabstop>
  <tabstop>nrml_scl</tabstop>
  <tabstop>filter_constant</tabstop>
  <tabstop>avg_mode</tabstop>
  <tabstop>window_mode</tabstop>
  <tabstop>restoreDefaultEQ</tabstop>
  <tabstop>eq_01</tabstop>
  <tabstop>eq_02</tabstop>
  <tabstop>eq_03</tabstop>
  <tabstop>eq_04</tabstop>
  <tabstop>eq_05</tabstop>
  <tabstop>eq_06</tabstop>
  <tabstop>eq_07</tabstop>
  <tabstop>eq_08</tabstop>
  <tabstop>eq_09</tabstop>
  <tabstop>eq_10</tabstop>
  <tabstop>eq_11</tabstop>
  <tabstop>eq_12</tabstop>
  <tabstop>eq_13</tabstop>
  <tabstop>eq_14</tabstop>
  <tabstop>eq_15</tabstop>
  <tabstop>eq_16</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
